src/main/java/org/opensearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -481,6 +482,7 @@ protected void successResponse(RestChannel channel) {
 		try {
 			final XContentBuilder builder = channel.newBuilder();
 			builder.startObject();
+			builder.endObject();
```

Comment:
Were we not calling `builder.startObject();` before ?

Reply:
No we were not doing it. That was a miss I guess.